lib/src/scss-shared/_nx-mask-helpers.scss

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ @import 'nx-colors';
+
+ @mixin background-mask($z-index) {
+  align-items: center;
+  background-color: transparentize($nx-grey-800, 0.25);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  overflow-y: hidden;
```

Comment:
Still not sure this property is necessary

Reply:
> Still not sure this property is necessary

Seems that it isn't, maybe a holdover from IE11 days? https://github.com/sonatype/sonatype-react-shared-components/pull/330/commits/c15fb39292d5247bd885798ff4bb8bd6700429eb